UnitTests/Input/KeyTests.cs

```diff
@@ -53,6 +53,59 @@ public void Constructor_Char (char ch, KeyCode expectedKeyCode)
 		Assert.Equal (expectedKeyCode, key.KeyCode);
 	}
 
+
+	// TryParse
+	[Theory]
+	[InlineData ("a", KeyCode.A)]
+	[InlineData ("Ctrl+A", KeyCode.A | KeyCode.CtrlMask)]
+	[InlineData ("Alt+A", KeyCode.A | KeyCode.AltMask)]
+	[InlineData ("Shift+A", KeyCode.A | KeyCode.ShiftMask)]
+	[InlineData ("A", KeyCode.A | KeyCode.ShiftMask)]
+	[InlineData ("â", (KeyCode)'â')]
+	[InlineData ("Shift+â", (KeyCode)'â' | KeyCode.ShiftMask)]
```

Comment:
This is wrong. 'â' is the lower case of 'Â'. So it haven't the shift key. Only the 'Â' has the shift key. Because of only consider the A..Z for the test the logic of the shift key is broken.